lib/ruby_indexer/lib/ruby_indexer/configuration.rb

```diff
@@ -273,7 +273,13 @@ def initial_excluded_gems
       end
 
       others.concat(this_gem.to_spec.dependencies) if this_gem
-      others.concat(others.filter_map { |d| d.to_spec&.dependencies }.flatten)
+      others.concat(
+        others.filter_map do |d|
+          d.to_spec&.dependencies
+        rescue Gem::MissingSpecError
+          false
```

Comment:
Any reason for `false` here rather than `nil`? I know the overall result is the same, but `nil` seems like clearer way to represent absence.